test/data/signet/node-defaults.yaml

```diff
@@ -0,0 +1,6 @@
+image:
+  repository: bitcoindevproject/bitcoin
+  pullPolicy: IfNotPresent
+  tag: "27.0"
+
+chain: signet
```

Comment:
I'd suggest updating the default example to explicitly have the `chain: regtest` argument specified. While testing, I had to come back to your PR to remember where to put the chain field.

Reply:
isnt the default set in `charts/bitcoincore/values.yml` ?